internal/goglfw/internal_windows.go

```diff
@@ -93,11 +91,7 @@ type context struct {
 	getProcAddress     func(string) uintptr
 	destroy            func(*Window) error
 
-	wgl struct {
-		dc       _HDC
-		handle   _HGLRC
-		interval int
-	}
+	state platformContextState
```

Comment:
`platform` sounds better than `state` as this represents a platform-specific state. What do you think?

Reply:
The thing is a state though which just happens to run on a specific platform. I think state is more clear to what it is while platform describes how is ran. I prefer state but if you want platform that's fine by me.

Reply:
Hmm, IMO `state` doesn't represent anything since most member variables are 'states' in general... So, I would like to go with `platform`